test/redteam/providers/simulatedUser.test.ts

```diff
@@ -0,0 +1,228 @@
+import { SimulatedUser } from '../../../src/providers/simulatedUser';
+import RedteamSimulatedUserProvider from '../../../src/redteam/providers/mischievousUser';
+import type { CallApiContextParams, CallApiOptionsParams, Prompt } from '../../../src/types';
+
+jest.mock('../../../src/providers/simulatedUser');
+
+describe('RedteamSimulatedUserProvider', () => {
```

Comment:
This file name is out of date and descriptions are out of date

Reply:
Good catch. I'm going to remove this file; it should not have been merged in.